pkg/plugins/utils/cargo/main.go

```diff
@@ -0,0 +1,59 @@
+package cargo
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+)
+
+type InlineKeyChain struct {
+	// [A][S][C] Token specifies the cargo registry token to use for authentication.
+	Token string `yaml:",omitempty"`
+	// [A][S][C] HeaderFormat specifies the cargo registry header format to use for authentication (defaults to `Bearer`).
+	HeaderFormat string `yaml:"headerFormat,omitempty"`
```

Comment:
```suggestion
	HeaderFormat string `yaml:"headerformat,omitempty"`
```
All user facing configuration must be lowercase. It's a jsonschema constraint otherwise the IDE returns an error.